js/private/npm_utils.bzl

```diff
@@ -0,0 +1,46 @@
+"Utility functions for npm rules"
+
+def _bazel_name(name, version):
+    "Make a bazel friendly name from a package name and a version that can be used in repository and target names"
+    escaped = name.replace("/", "_").replace("@", "at_")
+    version_escaped = _normalize_version(version)
+    return "%s_%s" % (escaped, version_escaped)
+
+def _normalize_version(version):
+    "Make a bazel friendly version information"
+
+    # 21.1.0_rollup@2.70.2 becomes 21.1.0_rollup_2.70.2
+    return version.replace("@", "_")
```

Comment:
I have another example where in case of 2 peer dependencies they are concatenated with a `+`

e.g.
```
/mobx-react/5.4.3_mobx@5.10.1+react@17.0.1
/@babel/eslint-parser/7.17.0_@babel+core@7.14.3+eslint@7.12.1
```

Reply:
Good catch. Probably the case of two peer dependencies.

Reply:
Pushed a new commit for this.